WordPress/WordPressApi/WordPressComApi.m

```diff
@@ -146,6 +146,11 @@ - (WPJSONRequestOperation *)POST:(NSString *)URLString
 #if !defined(NS_BLOCK_ASSERTIONS)
 - (void)assertApiVersion:(NSString *)URLString
 {
+    // Skip asserts for non API calls
+    if ([URLString rangeOfString:WordPressComApiClientEndpointURL].location == NSNotFound) {
+        return;
+    }
```

Comment:
On example is when following a site entered via the menu's search bar. A HEAD request is made to ensure the site isn't a 404.  It uses the API as a matter of convenience.  The above conditional was added in the 5.6.1 branch, but is also reflected here.